sample/Sample.iOS/Renderers/CheckboxRenderer.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+
+using Foundation;
+using IS.Toolkit.XamarinForms.Controls;
+using Sample.IOS.Renderers;
+using UIKit;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.iOS;
+
+[assembly: ExportRenderer(typeof(ISCheckbox), typeof(CheckboxRenderer))]
+namespace Sample.IOS.Renderers
+{
+    public class CheckboxRenderer : ViewRenderer<ISCheckbox, UISwitch>
+    {
+        protected override void OnElementChanged(ElementChangedEventArgs<ISCheckbox> e)
+        {
+            base.OnElementChanged(e);
+
+            if (e.NewElement != null)
+            {
+                if (Control == null)
+                {
+                    UISwitch switchControl = new UISwitch();
```

Comment:
I dont want to use UISwitch on iOS.
I want the control used in the UITableView  of this video :
https://www.youtube.com/watch?v=J0owoXImGV4